babel.config.js

```diff
@@ -4,6 +4,7 @@ const path = require( 'path' );
 
 const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 const isBrowser = isCalypsoClient || 'true' === process.env.TARGET_BROWSER;
+const output_dir = process.env.CALYPSO_SDK_OUTPUT_DIR || '.';
```

Comment:
Nitpick but let's use camelCased variable names. :-) Perhaps `sdkOutputDir` for added clarity?